File: src/js/fizzbuzz.js
#!/usr/bin/env nodejs

var num_to = 1000000

for(i = 1 ; i <= num_to; i++) {
    var out = "";
    if(i % 3 == 0) { out += "Fizz" }
    if(i % 5 == 0) { out += "Buzz" }
    if(out == "") { out = i }
    console.log(out)
}

File: src/c/fizzbuzz.c
#include <stdio.h>

int main(void)
{
    int i;
    for(i=1; i<=1000000; ++i)
    {
        if (i % 3 == 0)
            printf("Fizz");
        if (i % 5 == 0)
            printf("Buzz");
        if ((i % 3 != 0) && (i % 5 != 0))
            printf("%d", i);
        printf("\n");
    }

    return 0;
}
